app/src/main/java/org/mozilla/fenix/search/SearchFragment.kt

```diff
@@ -160,7 +160,11 @@ class SearchFragment : Fragment(), UserInteractionHandler {
             BrowserToolbar.Button(
                 ContextCompat.getDrawable(requireContext(), R.drawable.ic_microphone)!!,
                 requireContext().getString(R.string.voice_search_content_description),
-                visible = { requireContext().settings().shouldShowVoiceSearch && speechIsAvailable() },
+                visible = {
+                    requireContext().settings().shouldShowVoiceSearch &&
+                        speechIsAvailable() &&
+                        currentSearchEngine.searchEngine.name == "Google"
```

Comment:
Can we use the `identifier` instead? Does name work on other locales?

Reply:
We have multiple identifiers that refer to Google, so I don't think that works as well as just using the name. I guess I could just check for whether or not the identifier **contains** `google`?

Reply:
contains `google` sounds good, I'd prefer it to the name check